geo-types/Cargo.toml

```diff
@@ -20,7 +20,7 @@ use-rstar_0_9 = ["rstar_0_9", "approx"]
 
 [dependencies]
 approx = { version = ">= 0.4.0, < 0.6.0", optional = true }
-arbitrary = { version = "=1.1.3", optional = true }
+arbitrary = { version = "=1.2.2", optional = true }
```

Comment:
Yeah, I figured this would cause issues. I thought we had a MSRV policy, but I can't find it any more :-).

Reply:
I have have revert this, here is what cargo outdated returns now... 
not great but smaller.

```
geo-types
================
Name                Project  Compat   Latest   Kind    Platform
----                -------  ------   ------   ----    --------
arbitrary           1.1.3    ---      1.2.2    Normal  ---
as-slice            0.1.5    ---      Removed  Normal  ---
atomic-polyfill     0.1.11   Removed  ---      Normal  cfg(target_arch = "avr")
autocfg             1.1.0    Removed  ---      Build   ---
critical-section    1.1.1    Removed  ---      Normal  ---
generic-array       0.12.4   ---      Removed  Normal  ---
generic-array       0.13.3   0.12.4   Removed  Normal  ---
generic-array       0.14.6   ---      Removed  Normal  ---
generic-array       0.14.6   0.12.4   Removed  Normal  ---
hash32              0.1.1    ---      0.2.1    Normal  ---
hash32              0.2.1    0.1.1    ---      Normal  ---
heapless            0.6.1    ---      0.7.16   Normal  ---
heapless            0.7.16   0.6.1    ---      Normal  ---
lock_api            0.4.9    Removed  ---      Normal  ---
pdqselect           0.1.0    ---      Removed  Normal  ---
rstar               0.8.4    ---      0.9.3    Normal  ---
rstar               0.9.3    0.8.4    ---      Normal  ---
rustc_version       0.4.0    Removed  ---      Build   ---
scopeguard          1.1.0    Removed  ---      Normal  ---
semver              1.0.16   Removed  ---      Normal  ---
spin                0.9.4    Removed  ---      Normal  cfg(target_arch = "x86_64")
stable_deref_trait  1.2.0    ---      Removed  Normal  ---
typenum             1.16.0   ---      Removed  Normal  ---
version_check       0.9.4    ---      Removed  Build   ---
version_check       0.9.4    Removed  Removed  Build   ---

geo-bool-ops-benches
================
Name       Project  Compat  Latest  Kind         Platform
----       -------  ------  ------  ----         --------
approx     0.3.2    0.5    1   0.5.1   Normal       ---
geo-types  0.6.2    0.7.8   0.7.8   Development  ---
```

Reply:
Keep in mind that a lot of these are optional and not enabled by default, but still show up in `cargo outdated`.